index.html

```diff
@@ -522,13 +521,13 @@ <h2>
       </div>
       <p>
         If the <a>application context</a>'s <a>active document</a>'s
-        [=Document/URL=] is not <a data-lt="within-scope-manifest">within
-        scope</a> of the <a>application context</a>'s manifest, the user agent
-        SHOULD show a prominent UI element indicating the [=Document/URL=] or
-        at least its <a>origin</a>, including whether it is served over a
-        secure connection. This UI SHOULD differ from any UI used when the
-        [=Document/URL=] is <a>within scope</a>, in order to make it obvious
-        that the user is navigating off scope.
+        [=Document/URL=] is not [=within manifest scope=] of the <a>application
```

Comment:
How strongly do you feel about renaming this? I deliberately named this "within scope" (the same name as the other definition), designed to be a sort of C++ overloading of the name.

So that anywhere else, I can write "if _x_ is within scope of _app manifest_", without having to clarify that I am invoking this definition, or the other one. (And link to the correct one.) I would prefer to be able to keep doing that, rather than having to update all of the other usages of this to "within manifest scope of _app manifest_".

Reply:
I see you updated some (maybe not all?) of the other references to distinguish "within the navigation scope of" and "within manifest scope of". Note that there are other references to this concept from other WICG documents (even if this is not properly exported), like Web Share and File Handling.

I would prefer that we don't have to add that verbosity to what I imagine are a huge amount of usages.

Reply:
> How strongly do you feel about renaming this? I deliberately named this "within scope" (the same name as the other definition), designed to be a sort of C++ overloading of the name.

The rename is so it's more clear when used in other specs - want to avoid people accidentally using the wrong one ("_URL_ within scope of _other URL_" VS "within manifest scope"). When used locally, we can still do "within scope".

Reply:
I'm glad you are :) But I'd still prefer for them not to have to be so verbose.

Context: I've written enough sentences like this one:

> If the application context's active document's URL is not within scope of the application context's manifest

That's already pretty hard to read. We would have to change it to:

> If the application context's active document's URL is not within **manifest** scope of the application context's manifest

I added this to allow us to be less verbose ("within scope of the manifest" as opposed to "within scope of the navigation scope of the manifest"). Having to say "within manifest scope of the manifest" is hardly better than just deleting this convenience wrapper and using the URL-based one each time.

I'm not too concerned about people accidentally using the wrong one, since if they do, it just means they'll be linking to a slightly wrong position in the page. The meaning is clear and unambiguous: you're either talking about being within scope of a URL or within scope of a manifest.

Reply:
Happy to change them to simpler ones... um... how about:

```
[=manifest/within scope of=]
[=URL/within scope of=]
```

Then it takes the complexity out of reading the sentences, but puts the problem on the spec editor... they just get displayed as "within scope of" and links to the right thing.

Reply:
That's great if you can pull it off. (Note: I'm not concerned about what the code looks like, just what the document looks like when rendered.) If we can preserve the overloading (at the text level), that would be awesome.

Reply:
actually, there was a sentence where "within scope of" didn't work...  so "[=X/within scope=]" works all round... updating.